<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8"/>
	<title>Research project (PhD)</title>
	<meta name="author" content="jibe-b"/>
	<link type="text/css" rel="stylesheet" href="/css/style.css"/>
	<meta name="copyright" content="CC-BY-SA"/>
	<meta name="date" content="2016"/>
<script src="/js/ASCIIMathML.js"></script>
</head>
<body>

<!--<nav>-->

<p><a href="/">Home</a>
<a href="https://jibe-b.github.io/lab-notebook/">Lab Notebook</a>
<a href="https://jibe-b.github.io/project/">Projects</a>
<a href="https://jibe-b.github.io/project/smd/">SMd syntax</a>
<a href="http://jibe-b.github.io/project/pso/">Physical Sciences ontology</a>
<a href="https://jibe-b.github.io/blog/">Blog</a>
<a href="https://jibe-b.github.io/aboutme.html">About</a>
<a href="/traffic-analytics">tracking</a></p>

<!--</nav>-->

<p><a href="https://github.com/jibe-b/jibe-b.github.io"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/e7bbb0521b397edbd5fe43e7f760759336b5e05f/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677265656e5f3030373230302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png"></a></p>

<p><iframe src="http://htmlpreview.github.io/?https://github.com/AurelienLourot/fork-me-on-gitlab/master/button.html?href=https://gitlab.com/jibe-b"
        frameborder="0" scrolling="0" width="200px" height="37px"></iframe></p>

<h1 id="researchproject">Research project</h1>

<h2 id="mainidea">Main idea</h2>

<p>Biochemical networks are higly non-linear and their evolution inform on fundamental characteristics aquisition in particular for the problem of drug resistance aquisiton.</p>

<p>Available databases include biochemical network datasets and phyllogenetic trees.</p>

<p>Combining both will enable to study biochemical networks both on the statics and evolutionary approches.</p>

<p>What are the models that reproduce the aquisition of complex biochemical networks through evolution?</p>

<p>In silico evolution models are available and will be studied.</p>

<p>Analysis of the models needed to interprete and reproduce the observed trajectories will be studied, in order to identify lack of experimental data and flaws of simulation models and provide experiments and theoretical models demands.</p>

<p>This general process will be applied to a set of biochemical processes for which:</p>

<ul>
<li>phyllogenitic relationships are available for a set of closely related species</li>
<li>data on the biochemical process and optionaly the expression levels (raw data) is available</li>
<li>display non-linear behaviour</li>
</ul>

<p>Expected outcomes are </p>

<h2 id="roadmap">Roadmap</h2>

<p>[ 2016 &#8230;&#8230;&#8230;&#8230;&#8230;. 2017 &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;. 2018 &#8230;&#8230;&#8230;&#8230;&#8230;&#8230; 2019 &#8230;&#8230;&#8230;. ]</p>

<p>[ LITERATURE REVIEW &#8211; LITERATURE REVIEW &#8211; LITERATURE REVIEW &#8211; LITERATURE REVIEW &#8211; LITERATURE REVIEW ]</p>

<p>[ &#8230;&#8230;.. DATA AgREgATION - DATA AgREgATION &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;. ]</p>

<p>[ &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230; OWN EXPERIMENT - OWN EXPERIMENT &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;. ]</p>

<p>[ &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230; DATA ANALYSIS &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;.. ]</p>

<p>[ &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230; MODEL ANALYSIS &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;. ]</p>

<p>[ &#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230;&#8230; DATA AND MODEL DEMANDS ]</p>

<h2 id="advancement">Advancement</h2>

<h3 id="literaturereview">Literature review</h3>

<ul>
<li><strong>Quantitative and logic modelling of gene and molecular networks</strong></li>
</ul>

<p>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4604653/</p>

<ul>
<li>The smallest chemical reaction system with bistability</li>
</ul>

<p>http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752&#8211;0509&#8211;3&#8211;90</p>

<ul>
<li>Emergence of switch-like behavior in a large family of simple biochemical networks</li>
</ul>

<p>https://arxiv.org/pdf/1104.2845.pdf</p>

<ul>
<li>metabolic networks - theoretical book, including bifurcation</li>
</ul>

<p>http://www.tbi.univie.ac.at/~raim/harvest/workshop.brno/references/steuer09.pdf</p>

<ul>
<li>Breaking an Epigenetic Chromatin Switch: Curious Features of Hysteresis in Saccharomyces cerevisiae Telomeric Silencing</li>
</ul>

<p>http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0113516</p>

<ul>
<li>Periodic, Quasi-periodic and Chaotic Dynamics in Simple Gene Elements with Time Delays (2016)</li>
</ul>

<p>http://www.nature.com/articles/srep21037</p>

<ul>
<li>mathematics of chemical reaction networks</li>
</ul>

<p>http://vcp.med.harvard.edu/papers/crnt.pdf</p>

<ul>
<li>Reduction of dynamical biochemical reactions networks in computational biology</li>
</ul>

<p>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3400272/</p>

<ul>
<li>Biojazz</li>
</ul>

<p>http://nar.oxfordjournals.org/content/43/19/e123.full.pdf</p>

<p>https://oss-lab.github.io/biojazz/</p>

<ul>
<li>infer biochemical networks</li>
</ul>

<p>http://bioinformatics.oxfordjournals.org/content/29/11/1416.short</p>

<ul>
<li>reconstruction of gene regulatory networks - big data</li>
</ul>

<p>https://link.springer.com/article/10.1007/s12065&#8211;013&#8211;0098&#8211;7</p>

<ul>
<li>refined measurements techniques</li>
</ul>

<p>http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0152648</p>

<ul>
<li>non-linearity and evolution</li>
</ul>

<p>http://msb.embopress.org/content/8/1/564.full</p>

<ul>
<li>in silico evolution of signaling networks (with non-linear properties)</li>
</ul>

<p>http://q-bio.org/w/images/7/75/Qbio2015_25.pdf</p>

<ul>
<li>in silico evolution of non-linear biochemical networks</li>
</ul>

<p>https://link.springer.com/chapter/10.1007/978&#8211;1&#8211;4614&#8211;3567&#8211;9_8</p>

<ul>
<li>Gene expression model (in)validation by Fourier analysis</li>
</ul>

<p>http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752&#8211;0509&#8211;4&#8211;123</p>

<ul>
<li>Parameter estimation in tree graph metabolic networks</li>
</ul>

<p>https://peerj.com/articles/2417/</p>

<h3 id="dataagregation">Data agregation</h3>

<ul>
<li>Pathway Commons</li>
</ul>

<p>http://www.pathwaycommons.org/</p>

<ul>
<li>PhylomeDB</li>
</ul>

<p>http://www.phylomedb.org/</p>

<p>https://www.bioconductor.org/packages/release/bioc/vignettes/pwOmics/inst/doc/pwOmics.pdf</p>

<h3 id="ownexperiments">Own experiments</h3>

<h3 id="dataanalysis">Data analysis</h3>

<h3 id="advancementofmodelanalysis">Advancement of model analysis</h3>

<h3 id="advancementofexperimentsandtheoreticaldemandsformalisation">Advancement of experiments and theoretical demands formalisation</h3>

<h2 id="expectedoutcomes">Expected outcomes</h2>

<h2 id="team">Team</h2>

<p>looking for a lab</p>

<p>or open collaboration in the frame of the NumBio Lab</p>

<h2 id="otherpossiblesubjectsofresearch">Other possible subjects of research</h2>

<ul>
<li><p>Operons https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2776167/ in particular as they exhibit non-linear behaviours http://rsif.royalsocietypublishing.org/content/5/Suppl_1/S29 https://www.ncbi.nlm.nih.gov/pubmed/17351004</p></li>
<li><p>evolutionary algorithms and architecture http://trr141.de http://www.aevol.fr</p></li>
<li><p>transition to multicellularity http://rsif.royalsocietypublishing.org/content/12/102/20140982.abstract</p></li>
<li><p>E-coli long term experiment http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002444</p></li>
</ul>

<h2 id="footer"><footer></h2>

<p>Jib√© <a href="https://twitter.com/jibe_jeybee">@jibe_jeybee</a> <a href="aboutme.html">About me</a></p>

<p>All contents on this site are under the <a href="https://creativecommons.org/licenses/by/4.0/">free licence CC-BY</a></p>

<!-- Piwik -->

<script type="text/javascript">
  var _paq = _paq || [];
  _paq.push(['trackPageView']);
  _paq.push(['enableLinkTracking']);
  (function() {
    _paq.push(['setTrackerUrl',  'https://api-30dd7a75689deeb73993750d9c11f656.jibe-b.sandcats.io']);
    _paq.push(['setSiteId', 1]);
    _paq.push(['setApiToken', 'uoFEiyqfdWMylpZmtst05-7Yeas3aUan3cbv9BF_loC']);
    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
    g.type='text/javascript'; g.async=true; g.defer=true; g.src='https://g5b30n2ptnc1cgscg8po.jibe-b.sandcats.io/embed.js'; s.parentNode.insertBefore(g,s);
  })();
</script>

<!-- End Piwik Code -->

<script async defer src="https://hypothes.is/embed.js"></script>

<!--</footer>-->

</body>
</html>
