<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8"/>
	<title>Research project (PhD)</title>
	<meta name="author" content="jibe-b"/>
	<link type="text/css" rel="stylesheet" href="/css/style.css"/>
	<meta name="copyright" content="CC-BY-SA"/>
	<meta name="date" content="2016"/>
<script src="/js/ASCIIMathML.js"></script>
</head>
<body>

<!--<nav>-->

<p><a href="/">Home</a>
<a href="https://jibe-b.github.io/lab-notebook/">Lab Notebook</a>
<a href="https://jibe-b.github.io/project/">Projects</a>
<a href="https://jibe-b.github.io/project/smd/">SMd syntax</a>
<a href="http://jibe-b.github.io/project/pso/">Physical Sciences ontology</a>
<a href="https://jibe-b.github.io/blog/">Blog</a>
<a href="https://jibe-b.github.io/aboutme.html">About</a>
<a href="/traffic-analytics">tracking</a></p>

<!--</nav>-->

<p><a href="https://github.com/jibe-b/jibe-b.github.io"><img style="position: absolute; top: 0; right: 0; border: 0;" src="https://camo.githubusercontent.com/e7bbb0521b397edbd5fe43e7f760759336b5e05f/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677265656e5f3030373230302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png"></a></p>

<p><iframe src="http://htmlpreview.github.io/?https://github.com/AurelienLourot/fork-me-on-gitlab/master/button.html?href=https://gitlab.com/jibe-b"
        frameborder="0" scrolling="0" width="200px" height="37px"></iframe></p>

<h1 id="researchproject">Research project</h1>

<h2 id="forewords">Forewords</h2>
<p>Here is a presentation of a long term, extensive research project. This is first of all an intellectual challenge aiming at knowing what could be done around biochemical pathways. My university training enables me to consider working on mathematical models of dynamical systems and I try to complete the overview of the interface between physics and biology, that I had during my master's degree, with more specialized recent knowledge on biochemical pathways.</p>
<p>In the frame of a PhD project, I consider delimitating a specific subproject and devolop it in full detail. Embedding this project in a wider frame will make it easy to identify how different scientific questions are related and how insights in a specific direction may inform another direction.</p>
<h2 id="general-questions">General questions</h2>
<p>Biochemical pathways are the bottleneck between genetic information and molecular function (as well as between molecular function and genetic information).</p>
<p>Their non linear properties are associated to function, population[Llamosi 2016] and evolutionary questions[Maharjan 2012]. In particular, the activation and regulation of biochemical pathways is ensured by topological properties such as redundance[Nguyen 2012], which make the multiple parameters of these networks have optimal states[Berkhout 2012, Malukulta 2016]. The modification of kinetics under controlled external parameters inform on the intervals of function of biochemical pathways[Otero-Muras 2014, Bajic 2014].</p>
<p>Such pathways are influenced by a certain level of genome and phenotype variation influencing or not function[Yanagida 2015]. The accumulation of neutral variations (with regard to function) may enable adaptation to conditions of external stress[Barrick 2014]. Hence, the evolutionary trajectories of the genes that are the basis of the biochemical networks inform on the evolvability of function after accumulation of variation[Hosseini 2015].</p>
<p>The phylogeny of species with respect to classes of biochemical pathways informs on the rules of evolution of biochemical networks and on the mapping between genotype and phenotype[Morrisson 2016].</p>
<p>Biochemical pathways are included in a &quot;modular&quot; network involving the whole organism[Muto 2013]. External conditions activate and influence biochemical pathways, which may result in high level influence on the organism's phenotype[Cheng 2014] (see section <em>PhD project questions</em>).</p>
<h2 id="phd-project-questions">PhD project questions</h2>
<h3 id="controll-ability-of-biochemical-pathways">Controll <em>ability</em> of biochemical pathways</h3>
<p>Biochemical pathways are:</p>
<ul>
<li>inherently non linear</li>
<li>coupled in network of pathways</li>
<li>probabilistic and subject to</li>
</ul>
<p>These properties leave the question of controllabity open. Pathway engineering enables to achieve specific goals such as production of biopharmaceuticals[Soons 2013] and influence phenotypes through pathway engineering[Fajas 2013].</p>
<p>The question of controllability leads to achievable goals and limitations[Villaverde 2016, Wang 2016].</p>
<p>Models of biochemical pathways exist for a vast number of different specific pathways and conditions[]. They are often valid in a limited scope. The analysis of experimental data as well as theoretical developments in the field of parameter optimization and model selection[Maiwald 2016] enable to improve biochemical pathways models[Bachmann 2012]. In a second step, these models may be used to study the controllability of biochemical pathways[Gates 2016].</p>
<p>Cancer could be linked to metabolism modification, with a loss of regulation of nutrient uptake by cells[Pavlova 2015]. The input-output models could be used to optimize cancer therapy through controlled diet.</p>

<!--
<h2 id="forewords">Forewords</h2>

<p>Here is a presentation of a long term, extensive research project.
This is first of all an intellectual challenge aiming at knowing what could be done around biochemical pathways.
My university training enables me to consider working on mathematical models of dynamical systems
and I try to complete the overview of the interface between physics and biology,
that I had during my master&#8217;s degree, with more specialized recent knowledge on biochemical pathways.</p>

<p>In the frame of a PhD project, I consider delimitating a specific subproject and devolop it in full detail.
Embedding this project in a wider frame will make it easy to identify how different scientific questions
are related and how insights in a specific direction may inform another direction.</p>

<h2 id="inanutshell">In a nutshell</h2>

<p>Biochemical pathways are the bottleneck between genetic information and molecular function
(as well as between molecular function and genetic information).</p>

<p>Their non linear properties are associated to function, population and evolutionary questions[Maharjan 2012].
In particular, the activation and regulation of biochemical pathways is ensured by topological properties such as redundance[Nguyen 2012],
which make the multiple parameters of these networks have optimal states[Berkhout 2012, Malukulta 2016].
The modification of kinetics under controlled external parameters inform on the intervals of function of biochemical pathways[Otero-Muras 2014, Bajic 2014].</p>

<p>Such pathways are influenced by a certain level of genome and phenotype variation influencing or not function[Yanagida 2015].
The accumulation of neutral variations (with regard to function) may enable adaptation to conditions of external stress[Barrick 2014].
Hence, the evolutionary trajectories of the genes that are the basis of the biochemical networks inform on the evolvability of function after accumulation of variation[Hosseini 2015].</p>

<p>The phylogeny of species with respect to classes of biochemical pathways informs on the rules of evolution of biochemical networks
and on the mapping between genotype and phenotype[Morrisson 2016].</p>
	
<p>(under construction) Mathematical models of biochemical networks require both theoretical developments and information extraction from experimental
data. Methodological requirements concern parameter optimization as well as model selection and model reduction.</p>

<p>Biochemical pathways are included in a &#8220;modular&#8221; network involving the whole organism[Muto 2013].
External conditions activate and influence biochemical pathways, which may result in high level influence on the organism&#8217;s phenotype[Cheng 2014]. </p>

<h2 id=" mycentersofinterest"> My centers of interest</h2>

<ul>
<li>how topology and kinetics of classes of biochemical pathways enable biological function</li>
<li>genotype and phenotype: identify faisability of mapping and fundamental limitations due to non linearity of biochemical pathways</li>
<li>how biochemical networks are influenced by external conditions</li>
<li>how biochemical networks evolve</li>
</ul>

<h2 id="openquestions">Open questions</h2>

<h3 id="preliminarywork">Preliminary work</h3>

<p>Questions around the available knowledge, experimental data and models about biochemical pathways:</p>

<ul>
<li><p>literature review</p></li>
<li><p>Do the coupling of phylogeny and biochemical pathway databases inform evolution of biochemical pathways? [Mano 2010]</p></li>
<li><p>Do ODE based models of biochemical pathway kinetics cover input-output relationship?[Berkhout 2012]</p></li>
<li><p>Do in silico evolution models with biochemical networks as genotype to phenotype mapping exhibit rigorous properties? [Feng 2015]</p></li>
<li><p>Biochemical pathways of <em>Saccharomyces cerevisiae</em> are well known, including genome-wide base. How faisible is it to take it as a reference for each step of this project and how realistic is it to have several references. </p></li>
</ul>

<h3 id="questionsontopologyandkineticsofbiochemicalnetworks:">Questions on topology and kinetics of biochemical networks:</h3>

<p>Focus as a first step (PhD project)</p>

<ul>
<li>How are biochemichal networks responding to external conditions (non linearity, input-output)?</li>
<li>How are biochemical networks regulated depending on the external conditions?</li>
</ul>

<p>Axis of research: development and optimization of ODE models of response of biochemical pathways to external conditions.</p>

<p>Possible application to effect of fasting (modification of external conditions) on biochemical pathways, with application to cancer.</p>

<h3 id="questionsonadaptabilityofbiochemicalpathwaysthroughgenomevariationunderexternalpressure:">Questions on adaptability of biochemical pathways through genome variation under external pressure:</h3>

<p>Secondary focus (PhD project)</p>

<ul>
<li>How are biochemical pathways tolerant to the supposed accumulation of genetic variation?[Feng 2015]</li>
<li>How does the supposed accumulation of genetic variation influence the adaptability of biochemical pathways to extreme/never met external conditions?</li>
</ul>

<h3 id="questionsrelatedtoexperimentalevolutioncontrolofexternalpressure">Questions related to experimental evolution (control of external pressure)</h3>

<p>Beyond the scope of a PhD project</p>

<p>Assuming a sufficient knowledge and a sufficient availability of models of biochemical pathways response of kinetics to external conditions.</p>

<ul>
<li>How to optimize the external (pressure) conditions to obtain the variation of biochemical pathways with acquisition of new function?</li>
</ul>

<h3 id="questionsrelatedtoinsilicoevolutionofrelaxedhypothesisonexternalpressure">Questions related to in silico evolution of (relaxed hypothesis on external pressure)</h3>

<p>Beyond the scope of a PhD project</p>

<ul>
<li>How do models of biochemical networks and the heuristics on their resistance to accumulation of variations inform the open question of the genotype to phenotype mapping? [Hosseini 2015]</li>
</ul>


### Different possible approaches

1. evolution of biochemical pathways, relation between non linear properties and genome variation (How is the phylogeny with regard to classes of biochemical pathways? 
-> evolvability of biochemical pathways by class

Note: This is a phylogenetic question that I would like to see studied by a biologist.

2. evolvability of pathways, effect of genome variation of non linear properties (Evolvability of biochemical networks, involving ):
-> In silico evolution focusing on biochemical pathways non linear properties/functions)

What properties are resistant to genome variation? What type of


Note: This is a long term goal, that is mainly a computational question.

3. activation/regulation of pathways by control of external environment




4. Effect of external conditions on evolvability of biochemical pathways.

Knowing how external conditions influence biochemical pathways (4),
knowing how genome variation modify 




# Reformulation


Biochemical pathways are one proxy to life.


I would like to become expert in the (non linear, here is what I find passionant) behaviour of chemical pathways.



I am interested in doing evolutionary system biology.


I love systems biology, and I would like to work of models (ODE, parameter optimization) about
the effect of external conditions (concentration in chemicals) on the kinetics of these networks.

I love in silico evolution.

SELECTION: I would like to investigate the effect of external conditions (selective pressure) on evolution of pathways (topology and kinetics)
VARIATION: as well as I would like to know more about the evolvability (sensibility to genome variation) of biochemical networks


**Sous le capot**


As there is available experimental data, I would like to mine in these database the phylogeny of species with regards to classes of biochemical pathways.
This would help know the genome variations that are likely or not to conserve complex biochemical pathways.

Knowing this, in silico models of evolution would inform on this heuristics about the genome variation conserving or losing / modifying biochemical pathways.


**Perturbation exterieure**

I would like to optimize parameters of models applied to experimental data of biochemical pathways under controlled external parameters
and study the effect of extrem external parameters on kinetics / activation / regilation. 
[](http://www.nature.com/articles/srep01417)

Indeed, it would be really interesting to study the genetic trajectory of the population under such a external conditions.
Let's say that I will try to be ready for doing this when it will be experimentally faisible.
'




-------------

Ok, one biiiiiiiiiiiig risk is to try to play on every parameter at a time.

One parameter at a time, but a lot of possible parameters to play with.

# param = 




# param = external  

http://evoprog.eu/

## looking at genome variation

## looking at variation of kinetics





### Independant preliminary work

- study pathways/kinetics and their non linear properties
- select one class of pathways with interesting properties
- identify species that have this pathway
- study phylogeny of the species family and identify conservation or 

### Lab project








[Biojazz 2015]


## State of the art






## Questions

Which classes of networks are conserved? What is the influence of non linear properties on evolvability of biochemical networks?





Several classes of models are available to characterize both the topology of the pathway and the kinetics of the reactions involved in them.

The activity, regulation and 







## PhD project


Parameter optimization of 


















Biochemical networks are higly non-linear and their evolution inform on fundamental characteristics aquisition in particular for the problem of drug resistance aquisiton.

Available databases include biochemical network datasets and phyllogenetic trees.

Combining both will enable to study biochemical networks both on the statics and evolutionary approches.

What are the models that reproduce the aquisition of complex biochemical networks through evolution?

In silico evolution models are available and will be studied.

Analysis of the models needed to interprete and reproduce the observed trajectories will be studied, in order to identify lack of experimental data and flaws of simulation models and provide experiments and theoretical models demands.

This general process will be applied to a set of biochemical processes for which:

- phyllogenitic relationships are available for a set of closely related species
- data on the biochemical process and optionaly the expression levels (raw data) is available
- display non-linear behaviour


-->

<!--
Expected outcomes are 

## Roadmap

|   | 2016 | 2017 | 2018 |  2019 |
:--|:----:|:----:|:----:|:-----:|
| LITERATURE REVIEW --  LITERATURE REVIEW --  LITERATURE REVIEW -- LITERATURE REVIEW -- LITERATURE REVIEW |||||
| DATA AGREGATION - DATA AGREGATION  |||||
|| OWN EXPERIMENT -  OWN EXPERIMENT   ||||
 ||| DATA ANALYSIS  |||
 ||||  MODEL ANALYSIS ||
 ||||| DATA AND MODEL DEMANDS |
[Roadmap]

-->

<h2 id="advancement">Advancement</h2>

<h3 id="literaturereview">Literature review</h3>

<p><iframe src="https://www.zotero.org/groups/biochemical-pathways-non-linearity-evolution/items" width="100%" height="700"></iframe></p>

<h3 id="dataagregation">Data agregation</h3>

<ul>
<li>Pathway Commons</li>
</ul>

<p>http://www.pathwaycommons.org/</p>

<ul>
<li>PhylomeDB</li>
</ul>

<p>http://www.phylomedb.org/</p>

<p>https://www.bioconductor.org/packages/release/bioc/vignettes/pwOmics/inst/doc/pwOmics.pdf</p>

<h3 id="ownexperiments">Own experiments</h3>

<h3 id="dataanalysis">Data analysis</h3>

<h3 id="advancementofmodelanalysis">Advancement of model analysis</h3>

<h3 id="advancementofexperimentsandtheoreticaldemandsformalisation">Advancement of experiments and theoretical demands formalisation</h3>

<h2 id="expectedoutcomes">Expected outcomes</h2>

<h2 id="team">Team</h2>

<p>looking for a lab</p>

<p>or open collaboration in the frame of the NumBio Lab</p>

<h2 id="otherpossiblesubjectsofresearch">Other possible subjects of research</h2>

<ul>
<li><p>Operons https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2776167/ in particular as they exhibit non-linear behaviours http://rsif.royalsocietypublishing.org/content/5/Suppl_1/S29 https://www.ncbi.nlm.nih.gov/pubmed/17351004</p></li>
<li><p>evolutionary algorithms and architecture http://trr141.de http://www.aevol.fr</p></li>
<li><p>transition to multicellularity http://rsif.royalsocietypublishing.org/content/12/102/20140982.abstract</p></li>
<li><p>E-coli long term experiment http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002444</p></li>
</ul>

<h1 id="dashboard">Dashboard</h1>

<h2 id="questionstasks">Questions / tasks</h2>

<p>Please visit <a href="https://gitlab.com/jibe-b/biochemical-pathways-non-linearity-evolution/issues">https://gitlab.com/jibe-b/biochemical-pathways-non-linearity-evolution/issues</a> and open an issue for any idea, question, remark. (unfortunately, gitlab doesn&#8217;t show in iframes, this will come through the gitlab API.)</p>

<p><iframe src="https://gitlab.com/jibe-b/biochemical-pathways-non-linearity-evolution/issues" width="100%" height="400px"></iframe></p>

<h2 id="zoterosharedlibrary">Zotero shared library</h2>

<p><iframe src="https://www.zotero.org/groups/biochemical-pathways-non-linearity-evolution#field-table" width="100%" height="400px"></iframe></p>

<h2 id="databasesparqlendpoint">Database (SPARQL endpoint)</h2>

<p>Knowledge base management: I will convert as much content from the issue management system (gitlab) and the library (keeping track of attribution to contributors) into a RDF graph. I will then analyse the content of this database to valorise it.</p>

<p>(server under maintenance (in order to have a secure database). Please try <a href="http://sparql.open-scientist.org">sparql.open-scientist.org)</a> or <a href="http://92.222.70.120:9999/blazegraph/sparql">http://92.222.70.120:9999/blazegraph/sparql</a> for updates )</p>

<p><iframe src="http://sparql.open-scientist.org" width="100%" height="400px"></iframe></p>

<h2 id="labnotebook">LabNotebook</h2>

<p><iframe width="100%" height="600" src="https://hypothes.is/stream?q=user:jibe%20tag:bipanole"></iframe></p>
<footer>

<hr />

<p>Jibé <a href="https://twitter.com/jibe_jeybee">@jibe_jeybee</a> <a href="aboutme.html">About me</a></p>

<p>All contents on this site are under the <a href="https://creativecommons.org/licenses/by/4.0/">free licence CC-BY</a></p>

<!-- Piwik -->

<script type="text/javascript">
  var _paq = _paq || [];
  _paq.push(['trackPageView']);
  _paq.push(['enableLinkTracking']);
  (function() {
    _paq.push(['setTrackerUrl',  'https://api-30dd7a75689deeb73993750d9c11f656.jibe-b.sandcats.io']);
    _paq.push(['setSiteId', 1]);
    _paq.push(['setApiToken', 'uoFEiyqfdWMylpZmtst05-7Yeas3aUan3cbv9BF_loC']);
    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
    g.type='text/javascript'; g.async=true; g.defer=true; g.src='https://g5b30n2ptnc1cgscg8po.jibe-b.sandcats.io/embed.js'; s.parentNode.insertBefore(g,s);
  })();
</script>

<!-- End Piwik Code -->

<script async defer src="https://hypothes.is/embed.js"></script>

<!--</footer>-->

</body>
</html>
